### ISSUE TITLE
Hamburger Menu Toggle not showing if breakpoint is set to Tablet Extra

### ISSUE THREAD

**Issue by lensbreak** [elementor/elementor#21003]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Since the latest update of Elementor + Pro the hamburger menu does not appear although the Mobile Dropdown option is on and a breakpoint is set to tablet extra (>1200px) full width on. The menu just keeps displaying the menu items as normal. If the breakpoint is set to tablet (>1024px), then everything works. Seems like there is a bug in regard with the "Additional Custom Breakpoints" that can be enabled in the stable features section. This happens on a plain and pure install with elementor and hello theme.

### Steps to reproduce

1. Enable Additional Custom Breakpoints
2. Create a menu
3. Enable hamburger menu (aside)
4. Set breakpoint to tablet extra (>1200px)
5. Check on mobile phone or a small screen, the menu still shows all items and no hamburger toggle

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

== Active Plugins ==
	Elementor
		Version: 3.10.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.10.1
		Author: Elementor.com

**Comment by websplash** [comment]
Experiencing the same issue here, please have a look into this

**Comment by websplash** [comment]
Downgrading to 3.9.0 fixes the issue

**Comment by NPCRy** [comment]
Bump. Having this issue as well. When can we expect a resolution?

**Comment by lensbreak** [comment]
@KingYes You just released a new elementor version but did not fix such a simple bug? Why? This is how people start to loose trust in elementor. As of my experience the past 3+ years we will have to wait 6 months to see such small bugs being fixed. It's a shame. I still do not understand elementors management.

**Comment by websplash** [comment]
@lensbreak @NPCRy 
Seems to work for me with the new update:
Elementor Pro - 3.10.3
Elementor - 3.10.2

**Comment by lensbreak** [comment]
Nope, still not working on latest versions if menu toggle is set to TABLET EXTRA.
I even created a new menu and tested on my plain elementor staging setup. Still broken!
As I said, no surprise if we wait now months to see this fixed.

**Comment by pthorup** [comment]
Same issue with having Tablet Extra breakpoint. I set the mobile dropdown breakpoint for Tablet Extra and it doesn't appear. I set it for tablet instead, then it appears for tablet view. So clearly you can't get the hamburger to come larger than 768px.

**Comment by halcollins** [comment]
Having the same issues... Any known fixes?

**Comment by vintageniftys** [comment]
Exactly the same issue - have set up staging and updated Elementor Pro to 3.10.3 (Elementor 3.10.2) and still the same issue. This means we cannot update the plugins past 3.9.2 and pro 3.9.0. Madness. Its not just the toggle issue - we have other settings that are used with the tablet extra breakpoint so this is critical.

**Comment by chenisra** [comment]
same issue..  Elementor 3.11.0

**Comment by Alipio123** [comment]
This is a Critical issue, this must be fix ASAP!!! 

Tablet extra breakpoint doesnt work in Nav Menu Widget!

**Comment by lensbreak** [comment]
Still not working, extra breakpoints are all useless for now and we can not update anything. Last tested with
- Elementor Version 3.11.1
- Elementor Pro Version 3.11.1

@KingYes When are you going to fix this? You can not let thousands of clients wait again half a year for this, really...

**Comment by lensbreak** [comment]
Still not working...

Elementor Version 3.11.2
Elementor Pro Version 3.11.2
Hello Theme Version: 2.6.1

By now they should have removed the Additional Custom Breakpoints from the "Stable Features" list as so many users have reported this issue (not only here).

**Comment by lensbreak** [comment]
@KingYes Another release, another disappointment you did not fix such a small but essential bug.

Tablet Extra breakpoint still not working:

Elementor Version 3.11.3
Elementor Pro Version 3.11.4
Hello Theme Version: 2.6.1

**Comment by cthollaway** [comment]
Having the same issue here also. This is a serious bug. Any idea when we can see a fix?

Elementor Version 3.11.3
Elementor Pro Version 3.11.4

**Comment by rally-lad** [comment]
Same issue here across multiple sites.

Hello Theme
Elementor 3.11.3
Elemenror Pro 3.11.4

**Comment by lensbreak** [comment]
@nicholaszein Thank you for your update on this. Please also also fix https://github.com/elementor/elementor/issues/21490 before we wait another 2 months for it..

**Comment by nicholaszein** [comment]
Hi @lensbreak,

please understand that we have a procedure to follow. I understand you're frustrated but we ask you to be patient.

Kind regards

**Comment by bengpengyap** [comment]
Thank you! Just in time for one of my projects due to ship this weekend! Now my "known issues" list is empty.

**Comment by nicholaszein** [comment]
Hello everyone!

Apologies for the delay, and thank you all for the feedback.

#### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.11.5**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
